src/dodal/devices/ipin.py

```diff
@@ -1,8 +1,11 @@
-from ophyd import Component as Cpt
-from ophyd import Device, EpicsSignalRO, Kind
+from ophyd_async.core import HintedSignal, StandardReadable
+from ophyd_async.epics.signal import epics_signal_r
 
 
-class IPin(Device):
+class IPinAsync(StandardReadable):
```

Comment:
Should: I think given that every device will be an `Async` one in the next month or so the word `Async` here is redundant, we can just remove it.

Reply:
ok

Reply:
fixed